### PR TITLE
Update the RPC calls seele node and synchronous debt structure to block in the table

### PR DESCRIPTION
**Pull request by lizhengwei88**
Update the RPC calls seele node and synchronous debt structure to block in the table

### CODE REVIEW

File: rpc/seele_rpc_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	rpc := NewRPC(address)
+	if rpc == nil {
+		return nil, errors.New("newRPC failed")
+	}
+	if err := rpc.Connect(); err != nil {
+		return nil, fmt.Errorf("rpc connect failed:%s", err.Error())
+	}
+	return rpc, nil
+}
+func TestCurrentBlock(t *testing.T) {
```
Comment (WTer) [inline rpc/seele_rpc_test.go]: add one blank line between "}" and func

File: rpc/seele_rpc_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	if err != nil {
+		t.Fatal("newRPC error:", err)
+	}
+	currentBlock, err := rpc.CurrentBlock()
+	if err != nil {
+		t.Fatal("GetPeersInfo failed:", err)
+	}
+	t.Log(currentBlock)
+}
+func TestGetBlockByHeight(t *testing.T) {
```
Comment (WTer) [inline rpc/seele_rpc_test.go]: add one blank line between "}" and func

Comment (lizhengwei88) [reply rpc/seele_rpc_test.go]: OK

File: rpc/seele_rpc_test.go (hunk 1 above)
Comment (lizhengwei88) [reply rpc/seele_rpc_test.go]: ok

File: database/types.go [hunk 3]
```diff
@@ -23,19 +23,42 @@ type DBSimpleTxInBlock struct {
...
+	ShardNumber     int                     `bson:"shardNumber"`
+}
+
+//Debt describle a transaction which stored in the database
+type Debt struct {
+	Hash        string `bson:"hash"`
+	TxHash      string `bson:"txhash"`
+	From        string `bson:"from"`
+	To          string `bson:"to"`
+	Block       uint64 `bson:"block"`
```
Comment (ypenghui6) [inline database/types.go]: Block height?

Comment (lizhengwei88) [reply database/types.go]: ok

**Pull request merged**